src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Resources/MemberMatch/MemberMatchResourceValidatorTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Linq;
+using Hl7.Fhir.ElementModel;
+using Hl7.Fhir.Model;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using Microsoft.Extensions.Primitives;
+using Microsoft.Health.Core.Features.Context;
+using Microsoft.Health.Fhir.Core.Configs;
+using Microsoft.Health.Fhir.Core.Extensions;
+using Microsoft.Health.Fhir.Core.Features;
+using Microsoft.Health.Fhir.Core.Features.Context;
+using Microsoft.Health.Fhir.Core.Features.Resources.MemberMatch;
+using Microsoft.Health.Fhir.Core.Features.Validation;
+using Microsoft.Health.Fhir.Core.Features.Validation.Narratives;
+using Microsoft.Health.Fhir.Core.Messages.MemberMatch;
+using Microsoft.Health.Fhir.Core.UnitTests.Features.Validation.Narratives;
+using Microsoft.Health.Fhir.Tests.Common;
+using NSubstitute;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Core.UnitTests.Features.Resources.MemberMatch
+{
+    public class MemberMatchResourceValidatorTests : NarrativeDataTestBase
+    {
+#if !Stu3
```

Comment:
For my own understanding, whats incompatible here? Is it code or behavior?

Reply:
It is behavior. For stu3, we do not get back the minimum cardinality error as status is not a required field - http://hl7.org/fhir/STU3/coverage.html